extensions/control-plane/api/management-api/management-api-test-fixtures/src/testFixtures/java/org/eclipse/edc/connector/controlplane/test/system/utils/Participant.java

```diff
@@ -458,11 +457,11 @@ public String getTransferProcessState(String id) {
      *
      * @param id transfer process id.
      */
-    public void suspendTransfer(String id, String reason) {
+    public void suspendTransfer(String id) {
         var requestBodyBuilder = createObjectBuilder()
                 .add(CONTEXT, createObjectBuilder().add(VOCAB, EDC_NAMESPACE))
                 .add(TYPE, "SuspendTransfer")
-                .add("reason", reason);
+                .add("reason", "any reason");
```

Comment:
I think that this is not needed parameter here as it not impact into the process, but you are right as this can be used downstream we could not change it without proper notification.